discordbot/stocks/government/gov_menu.py

```diff
@@ -351,15 +351,16 @@ def check(reaction, user):
         except asyncio.TimeoutError:
             for emoji in emoji_list:
                 await msg.remove_reaction(emoji, ctx.bot.user)
-            embed = discord.Embed(
-                description="Error timeout - you snooze you lose! 😋",
-                colour=cfg.COLOR,
-                title="TIMEOUT Stocks: Government (GOV) Menu",
-            ).set_author(
-                name=cfg.AUTHOR_NAME,
-                icon_url=cfg.AUTHOR_ICON_URL,
-            )
-            await ctx.send(embed=embed)
+            if cfg.DEBUG:
```

Comment:
I don't think that this should only be displayed if debugging is on since the user will wonder why the command stopped working.

Reply:
I've disabled the "you snooze you loose" because of 2 reasons:
- the user has visual feedback of the command not working when the emoji reaction buttons disappear
- when we use the bot in a channel that has multiple participants (or even if there's only one heavy user in a direct chat) the message may pop up after the user started using a different menu and thus the message is irrelevant

I hope this explains the rationale, but let me know and I can make a short video to illustrate what I mean

Reply:
Oh! I haven't thought about that issue. Then there's the option to append the timeout text to the menu message when it's triggered. But you're solution to it is good!